joomla/joomla-cms
# [5.3] Removing LegacyPropertyManagementTrait::setProperties() usage in Core

### Summary of Changes
The LegacyPropertyManagementTrait is deprecated and we are still using setProperties in some areas. This PR removes the usage of that method from all occurences in our core.


### Testing Instructions
Test registering a user and logging in and view a category view in the frontend. That should trigger all the code. There should be no difference between before and after applying this PR.


### Link to documentations
Please select:
